docs/Measuring_Resonances.md

```diff
@@ -65,36 +65,28 @@ the system that may damage the electronics.
 
 ## Software installation
 
-Note that resonance measurements and shaper auto-calibration require additional
-software dependencies not installed by default. First, you will have to run on
-your Raspberry Pi the following command:
+Because setting up the software for auto-calibration takes significant
+time and resources, it is not installed by default. Run the following
+script to setup `numpy` and `matplotlib` to get started:
 ```
-~/klippy-env/bin/pip install -v numpy
+~/klipper/scripts/setup-shaper-calibrate.sh
 ```
-to install `numpy` package. Note that, depending on the performance of the
+Note that, depending on the performance of the
 CPU, it may take *a lot* of time, up to 10-20 minutes. Be patient and wait
 for the completion of the installation. On some occasions, if the board has
-too little RAM, the installation may fail and you will need to enable swap.
+too little RAM, the installation may fail, and you will need to
+[enable swap](https://nebl.io/neblio-university/enabling-increasing-raspberry-pi-swap/).
 
-Next, run the following command to install the additional dependencies:
-```
-sudo apt install python-numpy python-matplotlib
-```
-
-Afterwards, check and follow the instructions in the
-[RPi Microcontroller document](RPi_microcontroller.md) to setup the
-"linux mcu" on the Raspberry Pi.
-
-Make sure the Linux SPI driver is enabled by running `sudo
-raspi-config` and enabling SPI under the "Interfacing options" menu.
+If you intend to connect the ADXL345 to your Raspberry Pi, check and follow the
```

Comment:
TBH, the docs don't describe other possibilities in details. It's on my TODO-list to write a short section describing the possibility of connecting the ADXL345 to the printer board with potential pitfalls and gotchas, but it doesn't exist yet :) And the way it is written now may leave the reader puzzled (Like, oh, do I? But what are other options? But the docs don't describe other possibility). If you want to make it explicit, maybe it's worth mentioning that this is the suggested setup? E.g.

If you intend to connect the ADXL345 to your Raspberry Pi, which is the suggested setup, ...

Reply:
I think I'll add that a pi is the suggested solution for a permanent installation with the possibility to use your primary mcu as well.